Firmware/includes/display.c

```diff
@@ -40,6 +45,10 @@ void display_init() {
 	framebuffer[2] = 0;
 	framebuffer[1] = 0;
 	framebuffer[0] = 0;
+
+	timer0_init();
```

Comment:
do we want to melt timer and display? in my Opinion it's more useful if we put all timer relatet things in to timer.c and call only display ```display_update()```
What is your point of view?